packages/app/src/specs/SpecsListBanners.vue

```diff
@@ -110,6 +110,22 @@
     </p>
     <RequestAccessButton :gql="props.gql" />
   </Alert>
+  <RecordBanner
+    v-else-if="showRecordBanner"
+    v-model="showRecordBanner"
+  />
+  <ConnectProjectBanner
+    v-else-if="showConnectBanner"
+    v-model="showConnectBanner"
+  />
+  <CreateOrganizationBanner
+    v-else-if="showCreateOrganizationBanner"
+    v-model="showCreateOrganizationBanner"
+  />
+  <LoginBanner
+    v-else-if="showLoginBanner"
+    v-model="showLoginBanner"
```

Comment:
Similar comment here.  Seems like we could push the `if` inside these components (or use a wrapper) and drive the presence/absence of these things from the state being tracked via the `v-model`.